stock_request_employee/models/stock_request_order.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+from odoo import api, fields, models
+
+
+class StockRequestOrder(models.Model):
+
+    _inherit = 'stock.request.order'
+
+    to_employee = fields.Boolean()
+    employee_id = fields.Many2one(
+        'hr.employee',
+    )
+
+    @api.onchange('employee_id')
+    def _onchange_employee(self):
+        self.change_childs()
+
+    @api.onchange('to_employee')
+    def _onchange_to_employee(self):
+        if self.to_employee:
+            self.location_id = self.env.ref(
+                'stock_request_employee.location_employee'
+            )
+            if self.env.user.employee_ids:
+                self.employee_id = self.env.user.employee_ids[0]
+        else:
+            self.employee_id = False
+            self.location_id = False
+        self.change_childs()
+
+    def change_childs(self):
+        res = super().change_childs()
+        if not self._context.get('no_change_childs', False):
+            for line in self.stock_request_ids:
+                line.employee_id = self.employee_id
+                line.to_employee = self.to_employee
+        return res
+
+    @api.model
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False,
```

Comment:
This is crazy, but this is the only way to update the context dict of a view. :disappointed: